recipes/osx-sysroot/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "osx-sysroot" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+outputs:
+  - name: sysroot_{{ cross_target_platform }}
+    version: {{ sdk_version }}
+    build:
+      noarch: generic
+    run_exports:
+      strong:
+        - __osx >={{ sdk_version }}
+    test:
+      commands:
+        - echo "hello world!"
+
+about:
+  home: https://github.com/conda-forge/macosx-sysroot-feedstock
+  summary: metapackage to help set osx sdk version
```

Comment:
SDK version and the deployment target are two different things. We have MACOSX_SDK_VERSION (cf specific var) and MACOSX_DEPLOYMENT_TARGET.

Reply:
this is fixed!